### PR TITLE
Added sql migrations for clothes & outfits

### PR DESCRIPTION
**Pull request by jmuru**

### CODE REVIEW

File: 0001_initial_schema.down.sql [hunk 1]
```diff
@@ -1,5 +1,9 @@
 BEGIN
 
-    CREATE TABLE 
+    DROP TABLE IF EXISTS Clothes
 
-END;
\ No newline at end of file
+    DROP TABLE IF EXISTS Outfits
+
+    COMMIT;
```
Comment (ahmedjahmi) [inline 0001_initial_schema.down.sql]: Are we sure we need the COMMIT command? I'm referencing 
image

from this site: [https://www.tutorialspoint.com/sql/sql-transactions.htm](url)

File: 0001_initial_schema.up.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        outfit_id INT NOT NULL,
+        created_by BIGINT UNIQUE NOT NULL,
+        created_at TIMESTAMP,
+        last_modified TIMESTAMP,
+        item_id BIGINT UNIQUE NOT NULL,
+        FOREIGN KEY (created_by) REFERENCES Users(id),
+        FOREIGN KEY (item_id) REFERENCES Clothes(id),
+    )
+
+COMMIT
```
Comment (ahmedjahmi) [inline 0001_initial_schema.up.sql]: Same question for COMMIT as in the other file

File: 0001_initial_schema.up.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+BEGIN
+
+    CREATE TABLE Clothes (
+        id SERIAL PRIMARY KEY,
+        asset_url VARCHAR ( 50 ) UNIQUE NOT NULL,
```
Comment (ahmedjahmi) [inline 0001_initial_schema.up.sql]: Do we need a constraint of 50 characters here? considering that sometimes urls are long.

File: 0001_initial_schema.up.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
+
+BEGIN
+
+    CREATE TABLE Clothes (
+        id SERIAL PRIMARY KEY,
```
Comment (ahmedjahmi) [inline 0001_initial_schema.up.sql:4]: I looked into the SERIAL keyword. It does not guarantee a UNIQUE id, so maybe we should add the UNIQUE keyword here as well?

File: 0001_initial_schema.up.sql [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        asset_url VARCHAR ( 50 ) UNIQUE NOT NULL,
+        category VARCHAR NOT NULL,
+        created_at TIMESTAMP,
+        created_by BIGINT UNIQUE NOT NULL,
+        brand VARCHAR,
+        FOREIGN KEY (created_by) REFERENCES Users(id),
+    )
+
+    CREATE TABLE Outfits (
+        id SERIAL PRIMARY KEY
```
Comment (ahmedjahmi) [inline 0001_initial_schema.up.sql:14]: Consider UNIQUE keyword

File: 0001_initial_schema.down.sql (hunk 1 above)
Comment (jmuru) [reply 0001_initial_schema.down.sql]: we can try it without it i can change

**Review by ahmedjahmi — APPROVED** [review]
looks good to me

**Pull request merged**